crm_phonecall/views/crm_lead_view.xml

```diff
@@ -10,7 +10,7 @@
                 <button class="oe_inline oe_stat_button" type="action"
                     name="%(crm_phonecall.crm_case_categ_phone_incoming0)d"
                     icon="fa-phone"
-                    context="{'default_opportunity_id': active_id, 'search_default_opportunity_id': active_id, 'default_partner_id': partner_id, 'default_duration': 1.0}">
+                    context="{'default_opportunity_id': id, 'search_default_opportunity_id': id, 'default_partner_id': partner_id, 'default_duration': 1.0}">
```

Comment:
Is `id` always available in this context? I would say this is more a problem of Odoo than the module itself, with the proper handling of `active_id`...

Reply:
I think it is available if there is not id it is because is "new" and then there won't be any phone call.

And of course, the problem comes from Odoo. It isn't the first time I get this same problem. Another way to fix it is by creating a method instead of using an action, what do you think?

Reply:
My fear is that we get an error using `id`, as it's a "magic" field and not always available depending on the web client. Yes, a method would be a safer option.

Reply:
Done using method. :wink: